### PR TITLE
tests: more k8s-exec-rejected debug output

### PR DESCRIPTION
**Pull request by danmihai1**
Print more information useful for debugging. Also, use a separate YAML file for this test, instead of reusing someone else's file.

Fixes: #8270

### CODE REVIEW

File: tests/integration/kubernetes/runtimeclass_workloads/k8s-policy-exec-rejected.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2018 Intel Corporation
```
Comment (sprt) [inline tests/integration/kubernetes/runtimeclass_workloads/k8s-policy-exec-rejected.yaml:2]:
```suggestion
# Copyright (c) 2018 Microsoft Corporation
```

Comment (danmihai1) [reply tests/integration/kubernetes/runtimeclass_workloads/k8s-policy-exec-rejected.yaml:2]: This is a copy of a file contributed by Intel, with minor changes.

Comment (fidencio) [reply tests/integration/kubernetes/runtimeclass_workloads/k8s-policy-exec-rejected.yaml:2]: I'd **also** add MSFT copyright then, but that's not mandatory / should be needed unless I find something else that would require a re-spin of the PR.

**Review by fidencio — APPROVED** [review]
lgtm, thanks @danmihai1!

File: tests/integration/kubernetes/runtimeclass_workloads/k8s-policy-exec-rejected.yaml (hunk 1 above)
Comment (danmihai1) [reply tests/integration/kubernetes/runtimeclass_workloads/k8s-policy-exec-rejected.yaml]: Good to know - thanks guys! I made this change while rebasing today.

**Review by sprt — APPROVED** [review]

**Pull request merged**